src/vs/github1s/notification.ts

```diff
@@ -9,21 +9,30 @@ const NOTIFICATION_STORAGE_KEY = 'GITHUB1S_NOTIFICATION';
 // Change this if a new notification should be shown
 const NOTIFICATION_STORAGE_VALUE = '20210212';
 
-const notification = {
+const getGitpodRepoUrl = () => window.location.pathname.match(/^\/[^\/]+\/[^\/]+/) ? `https://gitpod.io/#https://github.com${window.location.pathname.match(/^\/[^\/]+\/[^\/]+/)![0]}` : '';
+
+const notifications = [{
 	title: 'ATTENTION: This page is NOT officially provided by GitHub.',
 	content:
 		'GitHub1s is an open source project, which is not officially provided by GitHub.',
 	link: 'https://github.com/conwnet/github1s',
-};
+}, {
+	title: `<a class="notification-link" href="${getGitpodRepoUrl()}" alt="Gitpod ready-to-code" target="_blank"><img src="https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod"></a>`,
+	content:
+	'Click the button above to develop your project in the cloud.',
+}];
 
-const notificationHtml = `
-<div class="notification-main">
-	<div class="notification-title">${notification.title}</div>
-	<div class="notification-content">
-		${notification.content}
-		<a class="notification-link" href="${notification.link}" target="_blank">See more</a>
-	</div>
-</div>
+const notificationHtml = `${
```

Comment:
I suggest we use a different way to promote the link since the notification will be invisible after users click the `OK` button.
![image](https://user-images.githubusercontent.com/503123/110571973-5ffe6e80-810d-11eb-9eff-da6de9d0ca9a.png)

There are a couple of places I can imagine:
- The left bar 
![image](https://user-images.githubusercontent.com/503123/110572093-9c31cf00-810d-11eb-8d0d-def351b4a585.png)

- The command line interface 
![image](https://user-images.githubusercontent.com/503123/110572109-a48a0a00-810d-11eb-88f4-3f622379d4f0.png)

Reply:
I agree, the notification bar was the lowest hanging fruit 😄 . I like your suggestions, let me look into that tomorrow (it's getting late 😴 ) and update the PR. Unless of course you feel like working on the PR before I wake up :).

Thanks for your support and feedback, I appreciate it!

Reply:
I added a command in my dev environment.

Any idea if there's a way to trust the domain by default? I already added `*.gitpod.io` to `src/vs/workbench/contrib/url/browser/trustedDomains.ts` (and also tried with `gitpod.io`).

![image](https://user-images.githubusercontent.com/788827/110635335-f9169f00-8178-11eb-86b8-2be9ab62c30e.png)

Reply:
You can add trust domain here: https://github.com/conwnet/github1s/blob/master/src/vs/workbench/contrib/url/browser/trustedDomains.ts#L304

Reply:
I tried the command on https://github1s-git-fork-mikenikles-integrate-with-gitpod-4fd92e.vercel.app/conwnet/github1s/blob/HEAD/tsconfig.json and it works without the external website warning.

Reply:
I've just pushed 80df292 which does require the trusted domain the same way it currently happens in `sponsors.ts`.

Thanks so much for your hints and guidance! I decided to use the status bar rather than the left bar. We can see how this goes and decide whether an additional left side icon will be required. Does that sound like a plan?

Reply:
Sounds great, thank you very much!